game.html

```diff
@@ -22,25 +24,35 @@ <h1>Dopłyń na Hel z <i><a href="index.html">tripcity</a></i></h1>
     <main>
         <div id="buttons">
             <button><b>start</b></button>
-            <button>instrukcja</button>
+            <button id="btn__instruction">instrukcja</button>
+            <div id="modal__instruction" class="modal__instruction">
+                <div class="modal__instruction--content">
+                    <span class="close">&times;</span>
+                    <p>Twój bohater musi dopłynąć na Hel by uruchomić latarnię morską i wskazać drogę statkom.Twoim zadaniem jest ominięcie mew na morzu, uważaj zachowaj bezpieczny dystans od nich! Sterowanie odbywa się za pomocą strzałek: lewo/prawo/góra/dół</p>
```

Comment:
To jakoś nie po polsku.

>  uważaj zachowaj bezpieczny dystans od nich!

Może powinno być:
Twoim zadaniem jest ominięcie mew na morzu.  Uważaj! Zachowaj bezpieczny dystans.

Reply:
Moje propozycja:
Spróbuj przepłynąć Zatokę Gdańską i dotrzeć na Hel, by uruchomić latarnię morską i wskazać drogę statkom. Uważaj na mewy! Zachowaj bezpieczny dystans.
Poruszaj się za pomocą strzałek: lewo/prawo/góra/dół.